developer_guides/debugability/perf-counters/index.rst

```diff
@@ -0,0 +1,62 @@
+.. _dbg-perf-counters:
+
+Performance Counters
+####################
+
+Firmware can be configured to trace performance counters for each processing
+component. Each performance trace entry include: 
+
+- component UUID
+
+- peak platform & cpu timer ticks consumed during component's copy processing
+
+.. note::
+  Performance timestamp macros use both the platform timer and cpu timer in case
+  the latter is not always-running-timer.
+
+Performance Counters usage
+**************************
+
+Currently performance counters can be only enabled statically during FW build in
+one of two ways:
+
+#. Select "Performance counter" from "Debug" menu using make menuconfig
+
+#. Add "CONFIG_PERFORMANCE_COUNTERS=y" to specific FW config (e.g. tigerlake_defconfig)
+
+
+Once the the performance counters are enabled they are logged periodically for
+each active component with the pipeline period frequency.
```

Comment:
After you enable the performance counters, they are logged periodically for
each active component with the pipeline period frequency.